ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -1426,6 +1428,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-ipi
+- as: baremetal-ipi-ovn-ipv4-basecap-none-f28
```

Comment:
ci job are placed in alphabetical order, sounds like we should move this job a bit earlier.